lib/github_backend.rb

```diff
@@ -16,6 +16,15 @@ def initialize(_args = {})
     @logger.level = Logger::DEBUG unless ENV['RACK_ENV'] == 'production'
   end
 
+  def testable_repos
+    projects = JSON.parse(Faraday.get('https://team-api.18f.gov/public/api/projects/').body)
```

Comment:
Perhaps make this a class constant? Or at least part of it, so you can say `File.join(PUBLIC_TEAM_API, 'projects')`?